tests/reports/class-wc-tests-reports-orders-stats.php

```diff
@@ -806,8 +806,8 @@ public function test_populate_and_query_multiple_intervals() {
 				'net_revenue'             => $net_revenue,
 				'avg_items_per_order'     => $num_items_sold / $orders_count,
 				'avg_order_value'         => $net_revenue / $orders_count,
-				'num_returning_customers' => $orders_count,
-				'num_new_customers'       => 0,
+				'num_returning_customers' => 0,
+				'num_new_customers'       => 2,
```

Comment:
No need to change that, but I am curious, I noticed in some places we are using the `$new_customers` variable and in some others we are hardcoding `2`. Is there any reason for that?
```suggestion
				'num_new_customers'       => $new_customers,
```

Reply:
I directly added `2` to most of the areas I changed since some were listed and some weren't.  It was probably used before because the number of calculated new customers was much higher and dependent on number of orders.

Updated all these references to use `$new_customers` so we can quickly change this in the future if needed.

Reply:
:clap: